R/redcap-metadata-coltypes.R

```diff
@@ -371,12 +374,13 @@ redcap_metadata_internal <- function(
   d <-
     d_meta %>%
     dplyr::mutate(
+      dags        = (.dags & (.data$field_name == .record_field)),
       autonumber  = (.autonumber & (.data$field_name == .record_field)),
     ) %>%
     dplyr::mutate(
       response =
         dplyr::case_when(
-          autonumber                                          ~ paste0("col_integer()"                        , "~~record_autonumbering is enabled for the project"),
+          autonumber & !dags                                  ~ paste0("col_integer()"                        , "~~record_autonumbering is enabled for the project"),
```

Comment:
@pwildenhain, how do you feel about a line here that spells out the combo (of dag & autonumbering) suggests a character?

```r
autonumber &  dags ~ paste0("col_character()" , "~~record_autonumbering & DAGs are enabled for the project"),
```